generator-mediator-js/app/templates/packagingTemplate/_cp-mediators-into-pkg.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+mkdir -p targets/trusty/usr/share
+rm -rf targets/trusty/usr/share/*
+
+echo "Cloning base mediators..."
+git clone https://github.com/jembi/<%= appName %>.git targets/trusty/usr/share/<%= appName %>
```

Comment:
We can't be sure where this repo will be. Maybe we should rather copy the mediator files rather than cloning from git.